schemas/container.json

```diff
@@ -228,11 +249,17 @@
             "ruleComponentSequencingEnabled"
           ],
           "additionalProperties": false
+        },
+        "id": {
```

Comment:
Why do we need the property id? Where do we use it?

Reply:
This is a piggy back of another ticket to emit the property id. It is not used for multi-cdn